aws-api-payload.json

```diff
@@ -0,0 +1,209 @@
+{
+    "Cluster": {
+        "StepConcurrencyLevel": 1,
+        "TerminationProtected": false,
+        "BootstrapActions": [],
+        "LogUri": "s3://$ADG_S3_LOG_BUCKET/emr/aws-ingestion/",
+        "Tags": [
+            {
+                "Value": "Ignore",
+                "Key": "Persistence"
+            },
+            {
+                "Value": "dataworks platform",
+                "Key": "Owner"
+            },
+            {
+                "Value": "false",
+                "Key": "has_stub_broker"
+            },
+            {
+                "Value": "terraform",
+                "Key": "CreatedBy"
+            },
+            {
+                "Value": "True",
+                "Key": "SSMEnabled"
+            },
+            {
+                "Value": "development",
+                "Key": "Environment"
+            },
+            {
+                "Value": "true",
+                "Key": "peered_with_ucfs"
+            },
+            {
+                "Value": "aws-ingestion",
+                "Key": "Application"
+            },
+            {
+                "Value": "stub-broker",
+                "Key": "k2hb_data_source"
+            },
+            {
+                "Value": "dev-ingest-hbase-clone",
+                "Key": "Name"
+            }
+        ],
+        "Applications": [
```

Comment:
That section's interesting!  We shouldn't put versions in this bit; they should be derived from the ReleaseLabel; I guess we might find more of this where the `describe-clusters` exposes internal stuff that we don't strictly need.